Gemfile

```diff
@@ -8,7 +8,7 @@ gem 'rails', '~> 6.1.4'
 # Variables can be overridden for local dev in Gemfile-dev
 @hostdata_gem ||= { github: '18F/identity-hostdata', tag: 'v3.4.0' }
 @logging_gem ||= { github: '18F/identity-logging', tag: 'v0.1.0' }
-@saml_gem ||= { github: '18F/saml_idp', tag: 'v0.14.3-18f' }
+@saml_gem ||= { github: '18F/saml_idp', branch: 'oyk-forceauthn-support' }
```

Comment:
Update this once https://github.com/18F/saml_idp/pull/50 is merged in with the correct tag.

Reply:
Done!